packages/wonder-blocks-modal/components/two-column-modal.js

```diff
@@ -35,33 +42,73 @@ type Props = {
 export default class TwoColumnModal extends React.Component<Props> {
     static defaultProps = {
         onClickCloseButton: () => {},
+        smallSidebar: false,
     };
 
     render() {
+        const {sidebar, footer, contents, smallSidebar} = this.props;
+
+        const sidebarView = smallSidebar ? (
+            <Cell
+                smCols={4}
+                largeCols={1}
+                style={[styles.column, styles.bgSidebar, styles.bgSidebarSmall]}
+            >
+                {sidebar}
+            </Cell>
+        ) : (
+            <FlexCell style={[styles.column, styles.bgSidebar]}>
+                <View style={styles.sidebar}>{sidebar}</View>
+            </FlexCell>
+        );
+
         return (
-            <View style={styles.container}>
+            <View
+                style={[
+                    styles.container,
+                    !sidebar && styles.containerNoSidebar,
+                ]}
+            >
+                <Grid spec={GRID_MODAL_SPEC}>
+                    <Row flush>
+                        {sidebar && sidebarView}
+                        <FlexCell
+                            style={[
+                                styles.column,
+                                styles.bgContents,
+                                !sidebar && styles.bgContentsNoSidebar,
+                            ]}
+                        >
+                            <View
+                                style={[
+                                    styles.contents,
+                                    smallSidebar && styles.contentsSmall,
+                                    !!footer && styles.contentsHasFooter,
```

Comment:
Surprised to see a double-bang here when the line above doesn't have one. Is that a Flow typing thing?

Reply:
Oh - `smallSidebar` is a boolean and `footer` is potentially a node, or null, or whatever. I like trying to keep it clean so that I'm consistently using booleans - don't think Flow cares about this, TBH.